### ISSUE TITLE
NuGet packages have revision 'clearlydefined'

### ISSUE THREAD

**Issue by Teju-Manchenella** [clearlydefined/service#176]
All NuGet components in dev and a few NuGet components in prod have their revision set to 'clearlydefined'. Unsure if this is an issue in the service or crawler.

Example in dev
<img width="1634" alt="screen shot 2018-07-25 at 3 05 50 pm" src="https://user-images.githubusercontent.com/23349235/43230211-4cecd316-901c-11e8-8e34-95d73889a3cf.png">

Example in prod
<img width="1631" alt="screen shot 2018-07-25 at 3 06 06 pm" src="https://user-images.githubusercontent.com/23349235/43230216-4f444338-901c-11e8-906c-7ace20cd2596.png">

**Comment by jeffmcaffer** [comment]
@Teju-Manchenella Is this still a problem?

**Comment by Teju-Manchenella** [comment]
Yes this is still an issue. I tried reindexing and there was no success. I cannot see the data in dev and prod so I am unsure what the actual revision is in harvested data. Along with that I am unsure at what point the revision is getting set to 'clearlydefined'.

**Comment by Teju-Manchenella** [comment]
This was an issue with the crawler in nugetFetch. Dev had been crawled with buggy code so version was not being set properly, which caused the definition revision to be set to 'clearlydefined'. The code was fixed; however, dev data was never fixed. The buggy code in the crawler was fixed at some point (not sure when) but only after was it used in production. Due to this fact all the nuget data in production is fine except that one example I have above. Me and Gene deleted all the nuget data in dev and recrawled the top 30 packages. In production we deleted the nuget.versioning data (buggy component above) and repopulated azure search. The issue has now been resolved.
